tests/_create_bucket.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+import boto3
+
+
+def main() -> int:
+    resource = boto3.resource(
+        service_name="s3",
+        endpoint_url="http://minio:9000",
+        region_name="us-east-1",
+        aws_access_key_id=os.getenv("MINIO_ACCESS_KEY"),
```

Comment:
Tests just use MINIO, but why env var to be called MINIO_, It's actually `S3_ACCESS_KEY` and `S3_SECRET_KEY`

Reply:
I took this variables from current tests, but sure: https://github.com/frappe/frappe_docker/blob/c2b15431281e470678676bd6b1d9ac0288536850/tests/integration-test.sh#L141-L142 Fixed: https://github.com/vrslev/frappe_docker/commit/5e16ed89b6695eab7b486164c2e13b5cdf750de8